[BUG] `topic` cog broken

```
[2025-02-17 15:47:09] [ERROR] red: Package loading failed
Traceback (most recent call last):
  File "/redbot/.local/lib/python3.11/site-packages/redbot/core/core_commands.py", line 189, in _load
    await bot.load_extension(spec)
  File "/redbot/.local/lib/python3.11/site-packages/redbot/core/bot.py", line 1708, in load_extension
    lib = spec.loader.load_module()
          ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap_external>", line 605, in _check_name_wrapper
  File "<frozen importlib._bootstrap_external>", line 1120, in load_module
  File "<frozen importlib._bootstrap_external>", line 945, in load_module
  File "<frozen importlib._bootstrap>", line 290, in _load_module_shim
  File "<frozen importlib._bootstrap>", line 721, in _load
  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/redbot/data/cogs/CogManager/cogs/topic/__init__.py", line 3, in <module>
    from .topic import Topic
  File "/redbot/data/cogs/CogManager/cogs/topic/topic.py", line 5, in <module>
    class Topic(commands.Cog):
  File "/redbot/data/cogs/CogManager/cogs/topic/topic.py", line 11, in Topic
    def _is_valid_channel(self, channel: discord.abc.Messageable | discord.interactions.InteractionChannel | None):
                                                                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: module 'discord.interactions' has no attribute 'InteractionChannel'
```

Ref. #264

--- a/enforcer/enforcer.py
+++ b/enforcer/enforcer.py
@@ -49,7 +49,7 @@ class EnforcerCog(commands.Cog):
 
         self.config.register_guild(**default_guild_settings, force_registration=True)
 
-    def _is_valid_channel(self, channel: discord.abc.GuildChannel | None):
+    def _is_valid_channel(self, channel: "discord.guild.GuildChannel | None"):
         if channel is not None and not isinstance(channel, (discord.ForumChannel, discord.CategoryChannel)):
             return channel
         return False

--- a/markov/markov.py
+++ b/markov/markov.py
@@ -408,7 +408,7 @@ class Markov(commands.Cog):
         # Return true (i.e. should process message) if all checks passed
         return True
 
-    async def get_enabled_channels(self, guild: discord.Guild) -> list[discord.abc.GuildChannel]:
+    async def get_enabled_channels(self, guild: discord.Guild) -> "list[discord.guild.GuildChannel]":
         """Retrieve a list of enabled channels in a given guild"""
         # Retrieve and iterate over enabled channels in specified guild,
         # appending each channel to the list of enabled channels.

--- a/quotes/quotes.py
+++ b/quotes/quotes.py
@@ -187,7 +187,7 @@ class QuotesCog(commands.Cog):
         error_embed = discord.Embed(title="Error", description=error_msg, colour=await ctx.embed_colour())
         await ctx.send(embed=error_embed)
 
-    def _is_valid_channel(self, channel: discord.abc.GuildChannel | discord.abc.Messageable | None):
+    def _is_valid_channel(self, channel: "discord.guild.GuildChannel | discord.abc.MessageableChannel | None"):
         if channel is not None and not isinstance(
             channel,
             (

--- a/reactrole/reactrole.py
+++ b/reactrole/reactrole.py
@@ -27,7 +27,7 @@ class ReactRoleCog(commands.Cog):
 
         self.config.register_guild(**default_guild_settings)
 
-    def _is_valid_channel(self, channel: discord.abc.GuildChannel | None):
+    def _is_valid_channel(self, channel: "discord.guild.GuildChannel | None"):
         if channel is not None and not isinstance(channel, (discord.ForumChannel, discord.CategoryChannel)):
             return channel
         return False

--- a/report/report.py
+++ b/report/report.py
@@ -29,7 +29,7 @@ class ReportCog(commands.Cog):
 
         self.config.register_guild(**default_guild_settings)
 
-    def _is_valid_channel(self, channel: discord.abc.GuildChannel | None):
+    def _is_valid_channel(self, channel: "discord.guild.GuildChannel | None"):
         if channel is not None and not isinstance(channel, (discord.ForumChannel, discord.CategoryChannel)):
             return channel
         return False

--- a/timeout/timeout.py
+++ b/timeout/timeout.py
@@ -22,7 +22,7 @@ class Timeout(commands.Cog):
 
     # Helper functions
 
-    def _is_valid_channel(self, channel: discord.abc.GuildChannel | None):
+    def _is_valid_channel(self, channel: "discord.guild.GuildChannel | None"):
         if channel is not None and not isinstance(channel, (discord.ForumChannel, discord.CategoryChannel)):
             return channel
         return False

--- a/topic/topic.py
+++ b/topic/topic.py
@@ -8,7 +8,7 @@ class Topic(commands.Cog):
     def __init__(self):
         pass
 
-    def _is_valid_channel(self, channel: discord.abc.Messageable | discord.interactions.InteractionChannel | None):
+    def _is_valid_channel(self, channel: "discord.abc.MessageableChannel | discord.interactions.InteractionChannel | None"):
         if channel is not None and not isinstance(
             channel,
             (

--- a/verify/verify.py
+++ b/verify/verify.py
@@ -419,7 +419,7 @@ class VerifyCog(commands.Cog):
 
     # Helper functions
 
-    def _is_valid_channel(self, channel: discord.abc.GuildChannel | None):
+    def _is_valid_channel(self, channel: "discord.guild.GuildChannel | None"):
         if channel is not None and not isinstance(channel, (discord.ForumChannel, discord.CategoryChannel)):
             return channel
         return False

